### PR TITLE
{bio}[foss/2020a] biom-format v2.1.10 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by robqiao**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/6533c1aab2f169c0dd2851afa35ad7f2 for a full test report.

File: easybuild/easyconfigs/b/biom-format/biom-format-2.1.10-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+The BIOM file format (canonically pronounced biome) is designed to be
+ a general-use format for representing biological sample by observation
+ contingency tables. BIOM is a recognized standard for the Earth Microbiome
+ Project and is a Genomics Standards Consortium supported project.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://files.pythonhosted.org/packages/2b/8f/\
```
Comment (verdurin) [inline easybuild/easyconfigs/b/biom-format/biom-format-2.1.10-foss-2020a-Python-3.8.2.eb:28]: @robqiao are you sure you need that unusual `source_urls`? It fails with a control character error when I try a test build. For `PythonPackage` do you actually need `source_urls` to be defined at all?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/biom-format/biom-format-2.1.10-foss-2020a-Python-3.8.2.eb (hunk 1 above)
Comment (robqiao) [reply easybuild/easyconfigs/b/biom-format/biom-format-2.1.10-foss-2020a-Python-3.8.2.eb:28]: @verdurin you are right, `curl` failed the default pypi path but `eb` got a hold of the gzip. Hmm ...

**Comment by robqiao** [comment]
@verdurin have another try?

**Comment by robqiao** [comment]
@verdurin @smoors Can we merge this PR?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/f56198a2672550f85390294dca256564 for a full test report.

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Going in, thanks @robqiao!

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 9 out of 9 (1 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/547bb53355e3f402921d220922d68312 for a full test report.

**Pull request merged**